client/src/components/body/Reading/OrderedList.jsx

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable react/prop-types */
```

Comment:
can we move away from disabling eslint standards? kinda defeats the purpose of having it.

Reply:
Good callout

Reply:
@notcruz Agreed!